File: app/src/main/java/cn/blinkdagger/androidLab/utils/TimeUtil.java
package cn.blinkdagger.androidLab.utils;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

/**
 * Created by d on 2016/11/16.
 */

public class TimeUtil {


    public static final SimpleDateFormat DEFAULT_SDF = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault());

    /**
     * 获取当前时间的默认的格式化字符串
     */
    public static String getCuerentTimeString() {
        String time = DEFAULT_SDF.format(new Date(System.currentTimeMillis()));
        return time;
    }
    /**
     * 获取当前时间的默认的格式化字符串
     */
    public static String getCurrentTimeStringInFormat(long milliseconds) {
        String time = DEFAULT_SDF.format(new Date(milliseconds));
        return time;
    }

    /**
     * 获取时间的对应格式化字符串
     */
    public static String getTimeStringInFormat(long milliseconds, SimpleDateFormat format) {
        return format.format(new Date(milliseconds));
    }
}

File: app/src/main/java/cn/blinkdagger/androidLab/utils/TimeUtil.kt
package cn.blinkdagger.androidLab.utils

import java.text.SimpleDateFormat
import java.util.*

/**
 * Created by d on 2016/11/16.
 */
object TimeUtil {
    val DEFAULT_SDF = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault())
    /**
     * 获取当前时间的默认的格式化字符串
     */
    val cuerentTimeString: String
        get() = DEFAULT_SDF.format(Date(System.currentTimeMillis()))

    /**
     * 获取当前时间的默认的格式化字符串
     */
    fun getCurrentTimeStringInFormat(milliseconds: Long): String {
        return DEFAULT_SDF.format(Date(milliseconds))
    }

    /**
     * 获取时间的对应格式化字符串
     */
    fun getTimeStringInFormat(milliseconds: Long, format: SimpleDateFormat): String {
        return format.format(Date(milliseconds))
    }
}